frontend/src/components/GeyserStatsBox.tsx

```diff
@@ -19,16 +19,17 @@ export const GeyserStatsBox: React.FC<Props> = ({ name, value, children }) => {
 }
 
 const GeyserStatsBoxContainer = styled.div`
-  height: 72px;
-  ${tw`mr-5 font-roboto bg-paleBlue border border-lightGray rounded-sm p-3`}
+  ${tw`h-40px sm:mr-5 sm:bg-paleBlue sm:border sm:border-lightGray sm:rounded-sm sm:p-3 sm:h-72px`}
 `
 
 const GeyserStatsBoxLabel = styled.span`
-  ${tw`mb-1 flex font-light`}
+  ${tw`mb-1 flex font-light text-sm sm:text-base`}
 `
 
 const GeyserStatsBoxValueContainer = styled.div`
   ${tw`flex flex-row`}
 `
 
-const GeyserStatsBoxValue = styled.span``
+const GeyserStatsBoxValue = styled.span`
+  ${tw`text-sm sm:text-base`}
```

Comment:
We seem to use this a lot! Let's break this out into its own re-usable css and use the utility class

```
// styles.ts

const ResponsiveText = styled.css ${tw`text-sm sm:text-base`}
```
```
// GeyserStatsBox.tsx

const GeyserStatsBoxValue = styled.span`
${ResponsiveText}
`
```